### ISSUE TITLE
Create an organization in github for Choko

### ISSUE THREAD

**Issue by sebas5384** [recidive/choko#263]
We are building a community, and an ecosystem which probably are gonna feel more organized and official about the projects around choko, like prana (maybe not), official extensions, distributions, etc...

I'm not talking about third-parties.

Make's sense?

**Comment by jardix22** [comment]
Absolutely yes!

**Comment by recidive** [comment]
Yes, at some point we should move to an organization. Not a priority now though.

**Comment by sebas5384** [comment]
Fair.

**Comment by recidive** [comment]
I've created the chokojs organization. Are you ok with that name?

We need to build a team. For this I'd like to know what people are really interested in contributing. And better if they are interested in maintaining some Choko subsystem.

Initially I'd like to ask you for me being the "Benevolent Dictator" until we set the path the project will take in the long run. This would mean only me will be able to push to the master branch until we elect another "core committer". What do you think?

I'm not sure how the moving will happen. I think everything within the repository will be moved along, and we will just need to change local remotes and update links to the repositories from e.g. choko.org.

Let's discuss this a little bit before taking action.

**Comment by sebas5384** [comment]
While I'm thinking about this issue, here are some links I'm reading:
- http://oss-watch.ac.uk/resources/benevolentdictatorgovernancemodel
- http://oss-watch.ac.uk/resources/governancemodels
- http://producingoss.com/en/producingoss.html
- http://reagle.org/joseph/2010/gfc/chapter-6.html

**Comment by sebas5384** [comment]
I completely agree @recidive :+1: 

What if we do a manifest? so in that way we can align more easily as a community which shares the same (or at least the main ones) principles and values.

**Comment by recidive** [comment]
@sebas5384 do you have an example for a manifest?

**Comment by smcgive** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
0xKaepora, @yuligesec, @yekongxingle, @xyhhx, @andylinee, @witem, @nonseodion, @engn33r, @haipro287, @typistv, @iricehasan, @Troublor, @0xNithi, @anko9801, @lmanini, @CadeYu, @crrrrrw, @DavidHoenisch, @0xrjman, @crypt0ro

**Comment by smcgive** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
nkrulikovsky, @alarrieux, @gcleite, @snn20, @cryptokeenz, @jintao-h, @varun-doshi, @Chirag-Jani, @jamesmorgan, @JIEDAWA, @SantiagoGregory, @phpmac, @shallowclouds, @hongtaox, @nazariyv, @Los-had, @rvbansal, @grillem754, @Simon-Busch, @alekscp
